docs/source/pythonapi/deplete.rst

```diff
@@ -195,14 +195,24 @@ total system energy.
    helpers.FissionYieldCutoffHelper
    helpers.FluxCollapseHelper
 
-The :class:`openmc.deplete.IndependentOperator` uses inner classes subclassed 
+The :class:`openmc.deplete.IndependentOperator` uses inner classes subclassed
 from those listed above to perform similar calculations.
 
+The following classes are used to define transfer rates to model continuous
+removal or feed of isotopes during a depletion analysis.
+
+.. autosummary::
+   :toctree: generated
+   :nosignatures:
+   :template: myclass.rst
+
+   transfer_rates._TransferRates
```

Comment:
If a class is truly private (with leading underscore), we wouldn't put it in the documentation. In this case, since the `transfer_rates` property on `Integrator` is accessible by the user, it probably makes sense to not have this class be private.
```suggestion
   transfer_rates.TransferRates
```